tests/test_czml.py

```diff
@@ -329,6 +329,117 @@ def test_czml_add_orbit():
     assert repr(extractor.packets) == expected_doc
 
 
+@pytest.mark.skipif("czml3" not in sys.modules, reason="requires czml3")
+def test_czml_add_trajectory():
+    start_epoch = iss.epoch
+    end_epoch = iss.epoch + molniya.period
+
+    sample_points = 10
+    color = [255, 255, 0]
+
+    positions = [0.0, 1.0, 2.0, 3.0, 60.0, 4.0, 5.0, 6.0, 120.0, 7.0, 8.0, 9.0]
```

Comment:
Rather than specifying the positions directly like this, `add_trajectory` should receive a `CartesianRepresentation` with the positions, and an `epochs` list (an `astropy.time.Time` object with several values in it).

Reply:
Okay. You mean the format in the packet should be like this?
`"2012-04-30T12:00Z", 1.0, 2.0, 3.0,
  "2012-04-30T12:01Z", 4.0, 5.0, 6.0,
  "2012-04-30T12:02Z", 7.0, 8.0, 9.0`
I had used the seconds from start_epoch format earlier.

Reply:
The format in the packet is correct as it is, but the user should not introduce the plain seconds as it is written now. The method should take care of the formatting.

Reply:
Okay.

Reply:
Should we also take care of the format ? Or we can trust the user to enter points in ```CartesianRepresentation``` and epochs in ```Time (isot)``` formats?
I think we should take care of that by casting them.